gui/create-item.lua

```diff
@@ -201,6 +400,21 @@ function hackWish(unit)
    if not creatureok then return end
    mattype,matindex=getCreatureRaceAndCaste(creatureTable[3])
   end
+  if df.item_type[itemtype]=='CORPSEPIECE' then
+	bodpartok,bodypart=script.showListPrompt('Wish','What body part should it be?',COLOR_LIGHTGREEN,getCreaturePartList(mattype,matindex))
+	if bodypart == 1 then
+		corpsepieceGeneric = true
+	end
+	-- script.showListPrompt('Wish',dick,COLOR_LIGHTGREEN,{'shit','dick','fuck','bitch'})
```

Comment:
Please remove this

Reply:
oops lmfao

Reply:
i have a tendency to use swears while testing/figuring out functions and stuff